static/main.css

```diff
@@ -169,7 +169,7 @@ main.container {
   letter-spacing: normal;
 }
 
-.Events-Manager {
+.hyphenated-words-in-small-caps {
```

Comment:
Hi, @bingzhang, 

>Let's use `hyphenated-words-in-small-caps` convention for naming CSS classes.

In that message from the previous PR, I was referring to the naming convention and showing it by an example class name and not suggesting using this specific name. Sorry if it wasn't clear. 

Please look at some examples of naming CSS classes. Also, there are other CSS classes in main.css that don't have this naming convention and need to be updated. For example, the `Logout` CSS class name could be changed to `login-logout-button`, indicating the purpose of this class if the same class is being used for both buttons. Please share any further questions here. Thanks.